utils/build_swift/tests/expected_options.py

```diff
@@ -145,18 +145,7 @@
     'lto_type': None,
     'show_sdks': False,
     'skip_build': False,
-    'stdlib_deployment_targets': [
-        'macosx-x86_64',
-        'iphonesimulator-i386',
-        'iphonesimulator-x86_64',
-        'appletvsimulator-x86_64',
-        'watchsimulator-i386',
-        'iphoneos-armv7',
-        'iphoneos-armv7s',
-        'iphoneos-arm64',
-        'appletvos-arm64',
-        'watchos-armv7k'
-    ],
```

Comment:
The idea here is that we're taking this out of the argument parser and inferring it ourselves if there is nothing passed, right?

Reply:
Exactly. Rather than inferring the default stdlib deployment targets as a part of the argument parsing logic it makes more sense to set the default value to `None` if the user didn't explicitly pass the flag and infer the default value in a more suitable location.